src/test/java/io/confluent/connect/jdbc/source/JdbcSourceTaskUpdateTest.java

```diff
@@ -472,6 +551,14 @@ private void startTask(String timestampColumn, String incrementingColumn, String
   }
 
   private void startTask(String timestampColumn, String incrementingColumn, String query, Long delay) {
+    startTask(timestampColumn, incrementingColumn, query, delay, null, null);
+  }
+
+  private void startTask(String timestampColumn, String incrementingColumn, String query, Long delay, Long timestampInitial) {
+    startTask(timestampColumn, incrementingColumn, query, delay, timestampInitial, null);
+  }
+
+  private void startTask(String timestampColumn, String incrementingColumn, String query, Long delay, Long timestampInitial, Long incrementInitial) {
```

Comment:
Is `incrementInitial` an artifact of trying to do something similar for the incremental offset? It's currently unused, so we should remove it or add support for incremental offsets, too.